src/tasks/runstart.ts

```diff
@@ -178,7 +179,17 @@ export const RunStartQuest: Quest = {
     {
       name: "Chateau Desk",
       completed: () => get("_chateauDeskHarvested") || !get("chateauAvailable"),
-      do: () => visitUrl("place.php?whichplace=chateau&action=chateau_desk"),
+      do: (): void => {
+        visitUrl("place.php?whichplace=chateau&action=chateau_desk");
+        const juice_bar_items: Item[] = [
+          $item`clove-flavored lip balm`,
+          $item`ectoplasm <i>au jus</i>`,
+          $item`gremlin juice`,
```

Comment:
gremlin juice can actually be helpful in killing the LOV enforcer, so we do not want to autosell it (either if you have access to the Tunnel of Love, or as a general rule of thumb)